src/pybind/mgr/rbd_support/mirror_snapshot_schedule.py

```diff
@@ -330,7 +331,6 @@ class MirrorSnapshotScheduleHandler:
 
     lock = Lock()
     condition = Condition(lock)
-    thread = None
```

Comment:
Hit the following mypy error for all the handlers due to my changes to the handlers' `shutdown()` method,
```
rbd_support/module.py:18: note: In module imported here,
rbd_support/__init__.py:2: note: ... from here:
rbd_support/mirror_snapshot_schedule.py: note: In member "shutdown" of class "MirrorSnapshotScheduleHandler":
rbd_support/mirror_snapshot_schedule.py:351: error: Item "None" of "Optional[Thread]" has no attribute "is_alive"
rbd_support/mirror_snapshot_schedule.py:353: error: Item "None" of "Optional[Thread]" has no attribute "join"
Found 6 errors in 3 files (checked 32 source files)
```
I removed the error by making `thread` an instance attribute. I can also get rid of the error by adding the following line in `shutdown()` method
```
assert self.thread is not None
```

I'm not sure why the handlers have `thread` as a class attribute?